ui/packages/consul-ui/mock-api/v1/acl/auth-method/_

```diff
@@ -14,17 +27,45 @@ ${
       case 'oidc':
         config = {
           OIDCDiscoveryURL: `https://${fake.internet.ip()}:8443`,
+          OIDCDiscoveryCACert: `-----BEGIN CERTIFICATE-----${fake.internet.password(1357)}-----END CERTIFICATE-----`,
```

Comment:
Do we know the format of how these come out of consul in the API?

'Generally' they have returns in them (which might mean you don't need to do the hard wrapping in the CSS to show these). If they come out without the returns then don't worry!

Reply:
I assume the same way certs come from other endpoints. This is  why I'm mocking it out the same way.